src/navigation-plugin.tsx

```diff
@@ -61,6 +62,7 @@ export class NavigationPlugin extends KalturaPlayer.core.BasePlugin {
     this._activeCuePointsMap = this._getDefaultActiveCuePointsMap();
     this._itemsOrder = prepareItemTypesOrder(this.config.itemsOrder);
     this._itemsFilter = isEmptyObject(this.config.itemsOrder) ? itemTypesOrder : config.itemsOrder;
+    this._player.registerService('navigation', this);
```

Comment:
what props `timeline` uses from navigation plugin? can we extract only some methods instead of populate entire plugin as a service?